src/tests/users/signup.test.js

```diff
@@ -1,28 +1,94 @@
 const expect = require('expect')
 const { request } = require('../../utils/test')
-const { UserFactory } = require('../../utils/factories/')
+const { UserFactory, QualificationFactory } = require('../../utils/factories/')
+const Qualification = require('../../models/qualifications')
+const User = require('../../models/user')
 
 describe('signup', () => {
   const fakeUser = UserFactory.generate()
+  let users = []
+  let skills = []
+  let knowledge = []
+  before(async () => {
+    for (let x = 0; x <= 1; x++) {
+      let user = await UserFactory.generate()
+      user = await User.signup(user)
+      users.push(user)
+    }
+    for (let y = 0; y < 5; y++) {
+      const skill = await Qualification.create(
+        QualificationFactory.generateSkill()
+      )
+      skills.push(`"${skill._id}"`)
+      const knowledge1 = await Qualification.create(
+        QualificationFactory.generateKnowledge()
+      )
+      knowledge.push(`"${knowledge1._id}"`)
+    }
+  })
+
+  it('should make user a mentor', () => {
+    return request({
+      query: `
+        mutation {
+          updateUserById (_id: "${users[0]._id}", record: {
+            mentorshipCertified: true
+          }) {
+            record {
+              email
+            }
+          }
+        }
+      `
+    })
+      .expect((res) => {
+        expect(res.body).toHaveProperty('data.updateUserById.record')
+        expect(res.body.data.updateUserById.record.email).toStrictEqual(
+          users[0].email
+        )
+      })
+      .expect(200)
+  })
+
+  it('there is more than 0 mentor in database', () => {
+    return request({
+      query: `
+        query {
+          users (filter: { mentorshipCertified: true }) {
+            email
+          }
+        }
+      `
+    })
+      .expect((res) => {
+        expect(res.body).toHaveProperty('data.users')
+        expect(res.body.data.users.length).toBeGreaterThan(0)
+      })
+      .expect(200)
+  })
 
-  it('should create a new user', () => {
+  it('should create a new user with mentor and atleast 5 skills and 5 knowledge', () => {
```

Comment:
You should create a separate test for your changes and not modify the ones that already exists to fit your changes

Reply:
Also, your test doesn't check for this at least 5 skills and knowledge

Reply:
signup process got updated so I had to update signup test. Older was outdated and had older column names.

Reply:
it checks for 5 in signup static

Reply:
Again - you should create your own test for this and not modify the existing ones. Create your own test to verify the functionality you want and not modify this since this tests other aspects as well

Reply:
added old test back